### DISCUSSION TITLE
Optimize performance? APP_NICENESS?

### DISCUSSION THREAD

**Discussion by SawkeeReemo** [jlesage/docker-firefox#235] [Q&A]
Running docker-firefox in a container on my Synology NAS, I'm noticing that everything I click on has a decent amount of lag, and the loading of even light weight pages or using Bitwarden to fill passwords is very slow. I wanted to see if using the APP_NICENESS environmental variable might increase my performance.

Secondly, how do you run the required docker option `--cap-add=SYS_NICE` from a docker compose file? I tried both of the following, but my container failed to start:
```
command:
     - --cap-add=SYS_NICE
```
and
```
command:
     - cap-add=SYS_NICE
```
But neither seemed to be correct.

The error I'm receiving is:
`Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "cap-add=SYS_NICE": executable file not found in $PATH: unknown`

**Comment by jlesage** [comment]
Try to add the capability to your compose file like this:
```
cap_add:
  - SYS_NICE
```
